OpenSim/Simulation/Model/BushingForce.cpp

```diff
@@ -110,23 +110,14 @@ void BushingForce::extendConnectToModel(Model& aModel)
 {
     Super::extendConnectToModel(aModel); // base class first
 
-    string errorMessage;
+    // Look up the two bodies being connected by bushing by name in the
+    // model. TODO: use Connectors
     const string& body1Name = get_body_1(); // error if unspecified
     const string& body2Name = get_body_2();
```

Comment:
@aseth1 What's the word on @msdemers's suggestions for renaming body to frame (here and elsewhere)?